wa-integrations/calendly/server.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios');
+const dotenv = require('dotenv');
+const express = require('express');
+const { composeGetEventRequestAxiosConfig } = require('./src/calendly');
+const { composeSendMessageRequestAxiosConfig } = require('./src/tyntec');
+
+dotenv.config();
+const CALENDLY_TOKEN = process.env.CALENDLY_TOKEN;
+const FROM = process.env.FROM;
+const PORT = process.env.PORT || 3000;
+const TO = process.env.TO;
+const TYNTEC_API_KEY = process.env.TYNTEC_API_KEY;
+const WHATSAPP_TEMPLATE_ID = process.env.WHATSAPP_TEMPLATE_ID;
+const WHATSAPP_TEMPLATE_LANG = process.env.WHATSAPP_TEMPLATE_LANG;
+
+const app = express();
+app.use(express.json());
+
+app.post('/', async function (req, res) {
+    if (req.body.payload.rescheduled) {
+        res.sendStatus(204);
+        return;
+    }
+
+    const getEventRequest = composeGetEventRequestAxiosConfig(CALENDLY_TOKEN, req.body.payload.event);
+    const eventResponse = await axios(getEventRequest);
+
+    const request = composeSendMessageRequestAxiosConfig(
+        TYNTEC_API_KEY,
+        {
+            to: TO,
```

Comment:
```suggestion
            to: eventResponse.data. text_reminder_number, 
```
Co takhle - zkusit poslat WA "místo" SMS reminderu. S tím, že by tam mohl být podobně jako máš if req.body.payload.reschedule, tak něco jako if (!req.body.payload. text_reminder_number){} a try WA, pokud selže tak to hodí SMS skrz Conversations API?

Reply:
@ridlees, what about this https://github.com/tyntec/api-samples/compare/5ab8bfe..c5a3b1c ? I'd prefer not to the the SMS part as complicates the sample a lot. I'd mention the SMS possibility only in the *More* section...